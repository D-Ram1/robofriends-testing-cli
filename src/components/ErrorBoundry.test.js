import React, { Component } from 'react';
import { shallow } from 'enzyme';
import ErrorBoundary from './ErrorBoundry';


describe("Error Boundary works", () => {
	it('renders correctly', () => {
		expect(shallow(<ErrorBoundary />));
	})
})

