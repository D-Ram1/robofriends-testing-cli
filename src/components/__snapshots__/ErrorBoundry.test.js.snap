// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`componentDidCatch works renders correctly 1`] = `""`;

exports[`renders correctly 1`] = `""`;
